@stellar/design-system/src/components/Button/index.tsx

```diff
@@ -2,26 +2,47 @@ import React from "react";
 import { Loader } from "../Loader";
 import "./styles.scss";
 
-interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+/**
```

Comment:
This approach works and it's nice that it would be documented in one place. Bonus for being where the actual component is.

Reply:
Nice! I think this is great